README.rst

```diff
@@ -22,56 +23,74 @@ Installation
 
 
 
+The hosted whitelist
+--------------------
+
+The hosted whitelist can be found at `whitelist`_
+This white-list is maintained by the team of good peoples behind the `whitelist`_ 
+project.
+
 Complementary whitelist
 -----------------------
 
-Our tool allow us to link one or more file(s) to the system which will be used in complementary of our whitelist list.
+`UHBW`_ allows you to link one or more file(s) to the system which will be used as 
+complementary to the hosted `whitelist`_, which is downloaded and used by default.
 
 Special markers
 ---------------
 
-If you already used a whitelist list you already know that we generaly only list all domains we want to whitelist one by one.
+If you already have tried to use a whitelist, you'll probably know, that in 
+generally you can only add one domain or URL per line in a file, for which you 
+want to whitelist.
 
-It's also possible to do that with our whitelisting system but we can do more.
+With UHBW you can do this, but in addition to that tedious way, UHBW allows you 
+to use ``Regex``, :code:`RZD` and :code:`ALL`
```

Comment:
@spirillen it's 

```
:code:`REG`
```

Reply:
I did think of that, but for a new reader, what is REG? so I left it intentionally as Regex for the initial reading to point the attention to this link Regex = REG

But if you prefer the other, I'll change.

Reply:
Okay let's leave it like that!